### PR TITLE
Update ko.po

### PR DESCRIPTION
**Pull request by JSY1728**
Update Korean po file

p.m. 14:17 | South Korea

### CODE REVIEW

File: translations/ko.po [hunk 1]
```diff
@@ -1111,7 +1111,7 @@ msgstr "가격"
 #: src\commands\rush-duel.ts:184
 msgctxt "command-name"
 msgid "rush-duel"
-msgstr ""
+msgstr "러시 듀얼"
```
Comment (kevinlul) [inline translations/ko.po:1114]: Test is failing because Discord doesn't permit spaces in command names.

**Review by kevinlul — CHANGES_REQUESTED** [review]

File: translations/ko.po [hunk 2]
```diff
@@ -268,7 +268,7 @@ msgstr "오류: 덱이 비어 있습니다."
 
 #: src\events\message-ping.ts:45
 msgid "Free and open source _Yu-Gi-Oh!_ bot"
-msgstr ""
+msgstr "무료 오픈 소스 유희왕 봇"
```
[the remark this answers is not in the document]
Comment (JSY1728) [reply translations/ko.po]: sorry. i fix this.

"무료 오픈 소스 유희왕 봇" => "자유 오픈 소스 유희왕 봇"

File: translations/ko.po [hunk 3]
```diff
@@ -848,47 +848,47 @@ msgstr "패스워드"
 #: src\commands\search.ts:56
 msgctxt "command-option"
 msgid "konami-id"
-msgstr ""
+msgstr "코나미 ID"
```
Comment (kevinlul) [inline translations/ko.po:851]: The other test failure is from the space here.

Comment (JSY1728) [reply translations/ko.po]: ok. i fix this.

File: translations/ko.po [hunk 4]
```diff
@@ -848,7 +848,7 @@ msgstr "패스워드"
 #: src\commands\search.ts:56
 msgctxt "command-option"
 msgid "konami-id"
-msgstr "코나미 ID"
+msgstr "코나미ID"
```
Comment (kevinlul) [inline translations/ko.po:851]: Whoops, um, the capital letters too, Discord doesn't like that either 💀

Comment (JSY1728) [reply translations/ko.po]: i removed this.

#: src\commands\search.ts:56
msgctxt "command-option"
msgid "konami-id"
msgstr "코나미ID" => ""

**Review by kevinlul — APPROVED** [review]
Thank you! 😄

**Pull request merged**